Makefile

```diff
@@ -239,9 +239,7 @@ check:
 
 .PHONY: unused
 unused:
-# The -fields option generates false-positives for embedded mutex and
-# related types. See https://github.com/dominikh/go-unused/issues/18.
-	-unused -fields ./... | grep -v -E '(\.pb\.go:|/C:|_string.go:|parser/(yacc|sql.y)|_cgo|Mutex)'
+	-unused -fields ./... | grep -v -E '(\.pb\.go:|/C:|_string.go:|parser/(yacc|sql.y)|_cgo|rocksDBLog)'
```

Comment:
`unused` returns a non-zero exit code if there are any matches. The leading `-` tells make to ignore that exit code. There is an open issue for the `unused` tool about ignoring auto-generated files which would allow it to give us a clean exit code and eliminate the need for the `-`.